templates/default/elasticsearch.init.erb

```diff
@@ -0,0 +1,193 @@
+#!/usr/bin/env bash
```

Comment:
I would love to get rid of this file completely. I have couple of ideas:

* Using the init scripts for Ubuntu and Redhat/Centos from the main repository (.deb and .rpm packages provide them)
* Adding just a really minimal, stupid init script and encourage people provide their own (something really simple like [Nginx](http://wiki.nginx.org/RedHatNginxInitScript) has?